mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -7563,6 +8594,13 @@ struct ThreadwiseCopyV2RewritePattern
 
     auto sourceType = op.source().getType().cast<VectorType>();
     auto destType = op.dest().getType().cast<MemRefType>();
+    InMemoryDataOperation dataOpration = InMemoryDataOperation::Set;
+    auto dataOpAttr = op->getAttr("data_operation");
+    if (dataOpAttr) {
+      int64_t dataOp = dataOpAttr.template cast<IntegerAttr>().getInt();
+      if (dataOp == 1)
```

Comment:
`data_operation = 1` is the attribute specifically used in backward weight + XDLOPS + f32 path, right?

Reply:
data_operation = 1 mean using atomic add

Reply:
a comment would be nice.